src/webgpu/web_platform/util.ts

```diff
@@ -108,7 +108,13 @@ export function waitForNextFrame(
 
 export async function getVideoFrameFromVideoElement(
   test: Fixture,
-  video: HTMLVideoElement
+  video: HTMLVideoElement,
+  colorSpace: VideoColorSpaceInit = {
+    primaries: 'bt709',
+    transfer: 'bt709',
+    matrix: 'bt709',
+    fullRange: false,
+  }
```

Comment:
nit: I think this default value could be replaced with `getVideoColorSpaceInit('REC709')`